library/src/main/java/com/novoda/downloadmanager/ManagedDownloadMigrationService.java

```diff
@@ -0,0 +1,5 @@
+package com.novoda.downloadmanager;
+
+public interface ManagedDownloadMigrationService extends DownloadMigrationService {
```

Comment:
I don't think this should be a separate interface - we need to keep the public API as small as possible.

Reply:
I thought this was remuved @Mecharyry